pkg/machines/reducers.es6

```diff
@@ -221,6 +223,11 @@ function systemInfo(state, action) {
     case UPDATE_LIBVIRT_STATE: {
         return Object.assign({}, state, { libvirtService:  Object.assign({}, state.libvirtService, action.state) });
     }
+    case SET_LOGGED_IN_USER: {
+        const newState = Object.assign({}, state);
+        newState.loggedUser = action.payload.loggedUser;
+        return newState;
```

Comment:
Out of curiosity, why doesn't the usual pattern with `return Object.assign({}, state, { newprop: value })` work here?

(not a biggie)

Reply:
Fixed.